swan-cern/files/swan_computing_config.py

```diff
@@ -312,7 +316,10 @@ async def _init_spark(self):
         notebook_container = self._get_pod_container('notebook')
         username = self.spawner.user.name
 
-        cluster = self.spawner.user_options[self.spawner.spark_cluster_field]
+        cluster = 'none'
```

Comment:
Here actually we know that a spark cluster has been selected, otherwise this function would not even be executed (it is protected by `_spark_enabled()`, which already checks the field.

So we can just do `cluster = self.spawner.user_options[self.spawner.spark_cluster_field]`.

And below, we can remove:
```
        if cluster == 'none':
            return
```
I know this is not part of your changes, but I just realized we can optimize that part.